docs/docs/concepts/flows.mdx

```diff
@@ -483,6 +483,44 @@ flow is completed.
   scope: "flow"
 ```
 
+#### Slot value validation
+
+You can now write slot value validation directly in your flow yaml file without the need to use custom actions.
+Add a `rejections` property to your `collect_information` step to define the validation rules.
```

Comment:
I think the first sentence is written from the POV of an existing rasa user; instead we should write it from the POV of a new user that doesn't know about custom actions. I don't think it adds much to mention actions here (and could confuse people).
```suggestion
You can define slot validation rules directly in your flow yaml file by adding a `rejections` property under
any `collect_information` step.
```